cluster/gke/util.sh

```diff
@@ -166,6 +166,10 @@ function kube-up() {
     create_args+=("--additional-zones=${ADDITIONAL_ZONES}")
   fi
 
+  if [[ ! -z "${IMAGE_FAMILY}:-" ]]; then
```

Comment:
I think that you mean `${IMAGE_FAMILY:-}` (the `:-` is currently after the curly brace.  Once you fix that LGTM.